doc/nrf/ug_thread_tools.rst

```diff
@@ -37,22 +37,242 @@ Thread Border Router
+               To install a compatible version, execute the following commands:
+
+               .. code-block:: console
+
+                  sudo apt-get -y install libusb-1.0-0-dev sed
+                  pip3 install click crcmod ecdsa intelhex libusb1 piccata protobuf pyserial pyyaml tqdm pc_ble_driver_py pyspinel
+                  pip3 install -U --no-dependencies nrfutil==6.0.1
+                  export PATH="$HOME/.local/bin:$PATH"
+
+         #. Generate the RCP firmware package:
+
+            .. code-block:: console
+
+               nrfutil pkg generate --hw-version 52 --sd-req=0x00 \
+                --application build/bin/ot-rcp.hex --application-version 1 build/bin/ot-rcp.zip
+
+         #. Connect the nRF52840 Dongle to the USB port.
+         #. Press the **RESET** button on the dongle to put it into the DFU mode.
+            The LED on the dongle starts blinking red.
+         #. Install the RCP firmware package onto the dongle by running the following command, with */dev/ttyACM0* replaced with the device node name of your nRF52840 Dongle:
+
+            .. parsed-literal::
+               :class: highlight
+
+               nrfutil dfu usb-serial -pkg build/bin/ot-rcp.zip -p */dev/ttyACM0*
+
+      .. tab:: nRF52840 Development Kit (UART transport)
+
+         a. Flash the image using the nrfjprog utility:
+
+            .. code-block:: console
+
+               nrfjprog -f nrf52 --chiperase --program build/bin/ot-rcp.hex --reset
+
+         #. Disable the Mass Storage feature on the device, so that it does not interfere with the core RCP functionalities:
+
+            .. code-block:: console
+
+               JLinkExe -device NRF52840_XXAA -if SWD -speed 4000 -autoconnect 1 -SelectEmuBySN *SEGGER_ID*
+               J-Link>MSDDisable
+               Probe configured successfully.
+               J-Link>exit
+
+            Replace *SEGGER_ID* with the SEGGER ID of your nRF52840 Development Kit.
+            This setting remains valid even if you flash another firmware onto the device.
+         #. Power-cycle the device to apply the changes.
+
+Configuring using Docker
+========================
+
+For the development purposes, you can run the OpenThread Border Router on any Linux-based system using a Docker container that already has the Border Router installed.
+This solution is better when you are only interested in direct communication between your Border Router and the Thread network.
+For example, you can use the Docker container when you want to establish IP communication between an application running on Linux (such as Python Controller for Matter) and an application running on a Thread node.
+
+If you go for the Docker container option, make sure to :ref:`configure radio co-processor <ug_thread_tools_tbr_rcp>` first.
+
+To install and configure OpenThread Border Router on any Linux-based operating system using the Docker container, complete the following steps:
```

Comment:
It looks like these steps are for Ubuntu/Debian, not "any Linux-based operating system"?

Reply:
That is right, in general, the OTBR can be installed on any Linux-based distribution, but we show how to achieve it on Ubuntu. I changed the wording.